Dockerfile.template

```diff
@@ -1,27 +1,25 @@
 # base-image for python on any machine using a template variable,
 # see more about dockerfile templates here: https://www.balena.io/docs/learn/develop/dockerfile/
-FROM balenalib/%%BALENA_MACHINE_NAME%%-python:3
+FROM balenalib/%%BALENA_MACHINE_NAME%%-python:3-run
```

Comment:
Is there a `3-stretch-run` to set the Debian version as well, just like in the simple-server-node project?